File: scratch/dump.py
import json
import urllib.request
import urllib.error

url = 'https://ezazhutkkntbfqpsweyu.supabase.co/rest/v1/employees?select=*'
req = urllib.request.Request(
    url,
    headers={
        'apikey': 'sb_publishable_Y-wTcUG5CF_oi93BcQsisQ_aOJipc3M',
        'Authorization': 'Bearer sb_publishable_Y-wTcUG5CF_oi93BcQsisQ_aOJipc3M'
    }
)

try:
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        parsed = json.loads(html)
        with open('scratch/employees_dump.json', 'w', encoding='utf-8') as f:
            json.dump(parsed, f, ensure_ascii=False, indent=2)
        print('Successfully wrote employees to scratch/employees_dump.json')
except urllib.error.URLError as e:
    print('URLError:', e.reason)
except Exception as e:
    print('Error:', str(e))

File: scratch/dump.js
const fs = require('fs');
const https = require('https');

const url = 'https://ezazhutkkntbfqpsweyu.supabase.co/rest/v1/employees?select=*';
const options = {
    headers: {
        'apikey': 'sb_publishable_Y-wTcUG5CF_oi93BcQsisQ_aOJipc3M',
        'Authorization': 'Bearer sb_publishable_Y-wTcUG5CF_oi93BcQsisQ_aOJipc3M'
    }
};

https.get(url, options, (res) => {
    let data = '';
    res.on('data', (chunk) => {
        data += chunk;
    });
    res.on('end', () => {
        try {
            const parsed = JSON.parse(data);
            fs.writeFileSync('scratch/employees_dump.json', JSON.stringify(parsed, null, 2), 'utf8');
            console.log('Successfully wrote employees to scratch/employees_dump.json');
        } catch (e) {
            console.error('Failed to parse JSON:', e.message);
            console.error('Raw data was:', data);
        }
    });
}).on('error', (err) => {
    console.error('HTTPS request error:', err.message);
});
